src/Css.elm

```diff
@@ -3236,6 +3242,58 @@ slashedZero =
   }
 
 
+{- FEATURE TAG VALUES -}
+
+
+{-| Alias for `on` value of [`font-feature-settings`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-feature-settings#Formal_syntax)
+-}
+on : Int
+on =
```

Comment:
So one of the things we found was that if you were trying to have styles alongside your HTML, using names like `on` would collide with things from elm-html. So this would likely become `Css.on` in most real code. I wonder if a longer, more qualified name, would be better, since then it won't collide? Especially for something that is rarely actually going to be used, because this will mean that I will have to use `Html.Events.on` too